tests/manual/test-procedure.rst

```diff
@@ -249,6 +249,96 @@ Code block 1::
     display.show(Image.PACMAN)
 
 
+Test Case: module.py file can be loaded by file picker and used in main.py
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- Load by Load/Save > Add file the file ``emoji.py``
+- [ ] Confirm the file shows up in the files list with the same title.
+- Return to the editor and replace the current script with the following:
+
+    from microbit import *
```

Comment:
Still didn't get formatted correctly, I think it needs the previous line with `::`.
![image](https://user-images.githubusercontent.com/29712657/61801131-2d174200-ae26-11e9-9882-f5fca8406d16.png)

Reply:
Sorry just noticed the previous line has the double colon.
I'd recommend to edit the doc from the GitHub interface, you get live previous that way to figure out how to make it work.